src/views/blocks/rsvp.php

```diff
@@ -24,9 +26,15 @@
 if ( ! $has_rsvps ) {
 	return false;
 }
-?>
 
-<?php $this->template( 'blocks/attendees/order-links' ); ?>
+$html = $this->template( 'blocks/attendees/order-links', [], false );
+
+if ( empty( $html ) ) {
+	$html = $this->template( 'blocks/attendees/view-link', [], false );;
+}
+
+echo $html;
```

Comment:
:warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$html'

<sub>hash: 634519d02589a2e2d676bc73f3c58284</sub>